lib/hammer_cli_foreman/associating_commands.rb

```diff
@@ -229,23 +229,23 @@ class RemoveUsergroupCommand < HammerCLIForeman::RemoveAssociatedCommand
       end
     end
 
-    module ConfigTemplate
+    module ProvisioningTemplate
       extend CommandExtension
 
-      class AddConfigTemplateCommand < HammerCLIForeman::AddAssociatedCommand
-        associated_resource :config_templates
-        desc _("Associate a configuration template")
+      class AddProvisioningTemplateCommand < HammerCLIForeman::AddAssociatedCommand
+        associated_resource :provisioning_templates
+        desc _("Associate a Provisioning template")
 
         success_message _("The configuration template has been associated.")
-        failure_message _("Could not associate the configuration template")
+        failure_message _("Could not associate the Provisioning template")
       end
 
       class RemoveConfigTemplateCommand < HammerCLIForeman::RemoveAssociatedCommand
```

Comment:
Should this also be renamed?
```suggestion
       class RemoveProvisioningTemplateCommand < HammerCLIForeman::RemoveAssociatedCommand
```